README.md

```diff
@@ -1,7 +1,15 @@
 # Northcoders News API
+In order to access the databases, create the files:
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+* #### .env.test
 
+* #### .env.development
+
+#### Inside .env.test add the line:
+    PGDATABASE=nc_news_test
+
+#### Inside .env.development add the line:
+    PGDATABASE=nc_news
 
```

Comment:
What is the purpose of a .env file and why should we add them to gitignore?

Reply:
Whoops! The purpose is to keep these private, the content of those files should be provided directly to the other developer in another way, or the other developer can create their own database name. I will adjust the readme file.